packages/complex-spaces.lua

```diff
@@ -0,0 +1,41 @@
+SILE.nodeMakers["x-spaces-are-nodes"] = pl.class({
+    _base = SILE.nodeMakers.unicode,
+    makeGlue = function (self, item)
+      self:addToken(" ", item)
+      self:makeToken()
+    end
+})
+
+SILE.languageSupport.languages["x-spaces-are-nodes"] = true
+
+SILE.settings.declare({
+  parameter = "shaper.complexspaces",
+  default = true,
+  type = "boolean",
+  help = "Whether the font's space glyph should be emitted, rather than a glue"
+})
+
+if SILE.shaper then
```

Comment:
The current module load tree is seriously borked –this test should be superfluous— but I'll refactor that when I finish fixing the class inheritance for shapers.

Reply:
I, um, only added that to keep the busted "can we load this package" thing happy.